include/nbl/asset/ICPUImage.h

```diff
@@ -175,8 +175,10 @@ class NBL_API ICPUImage final : public IImage, public IAsset
 		{
 			assert(!isImmutable_debug());
 
-			if (!IImage::validateCopies(_regions->begin(),_regions->end(),_buffer.get()))
+			if (!IImage::validateCopies(_regions->begin(),_regions->end(),_buffer.get())) {
```

Comment:
I like the ANSI C `{` convention (next line)

Reply:
Not intentional